modules/gsp-cluster/values.tf

```
@@ -39,5 +39,14 @@ data "template_file" "values" {
     grafana_default_admin_password                 = jsonencode(random_password.grafana_default_admin_password.result)
     eks_version                                    = var.eks_version
     cert_manager_role_arn                          = aws_iam_role.cert_manager.arn
+    dockerhub_username                             = var.dockerhub_username
+    dockerhub_password                             = var.dockerhub_password
+    dockerhub_credentials                          = jsonencode({
+        "auths" = {
+            "https://index.docker.io/v1/" = {
+                "auth" = base64encode(format("%s:%s", var.dockerhub_username, var.dockerhub_password))
+            }
+        }
+    })
```

I think we actually want to treat `dockerhub_credentials` as an opaque base64 blob here. otherwise it will end up as propper "YAML" in the values file. Alternatively, instead of trying to `b64enc` an object when creating the `Secret`, first encode it as a JSON string (which is possibly better), but either way I don't think it will thread through correctly as it is.

```suggestion
    dockerhub_credentials                          = base64encode(jsonencode({
        "auths" = {
            "https://index.docker.io/v1/" = {
                "auth" = base64encode(format("%s:%s", var.dockerhub_username, var.dockerhub_password))
            }
        }
    }))
```